public/js/reports/reports.service.js

```diff
@@ -79,5 +80,33 @@
 
             return columnDescription;
         };
+
+        function checkPrompts (prompts) {
```

Comment:
Can you add a comment to explain what this function checks exactly ?

Reply:
comment added